README.md

```diff
@@ -1,593 +1,93 @@
-# Get Started with Pieces OS and the @pieces.app/pieces-os-client npm package
+<h1 align="center">
+    <b>
+        <a href="https://pieces.app">
+            <picture>
+                <source srcset="./assets/Logo-light-theme.png" media="(prefers-color-scheme: light)">
+                <source srcset="./assets/Logo-dark-theme.png" media="(prefers-color-scheme: dark)">
+                <img src="./assets/Logo-dark-theme.png" height="125" width="600" />
+            </picture>
+        </a><br>
+    </b>
+</h1>
 
-Follow this guide to get an early look at using Pieces OS in your own development environment. You can use this to get familiar with some key terms and endpoints: `assets`, `asset`, `connect`, `format`, `formats`, and `create`. There are other topics that are touched on, and further expansion on this starter project is coming soon. 
 
-[![npm version](https://badge.fury.io/js/@pieces.app%2Fpieces-os-client.svg)](https://badge.fury.io/js/@pieces.app%2Fpieces-os-client)
+# <p align="center"> Example-Typescript
```

Comment:
Can we change this name to `React x Pieces` so that it's a bit more aligned with the project? Also make this edit everywhere? No need to change the repo title, just update the readme copy for it

Reply:
Added this